transcendence-app/src/users/users.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
+import { CreateUserDto } from './dto/create-user.dto';
+
+// This should be a real class/interface representing a user entity
+export type UserEntity = {
+  provider: string;
+  externalProviderId: number;
+  username: string;
+  email: string;
+  image_url: string;
+};
+
+@Injectable()
+export class UsersService {
+  private readonly users = [];
+
+  findOne(username: string) {
```

Comment:
This could be private method?

Reply:
Done

Reply:
Are you sure this method, and the 'create' one can be private?
I tried to create a 'user' controller and tried to call this method from there, like we are going to do when we want to see one user info or to create a new user.
We can change it later when the controller is ready.

Reply:
I think in the context of this PR, they can be private. If we add some routes to the user controller which use these functions, we can make them public again.

Reply:
Agree with @fizcamposancos for now I'd stay with private and, if we need to use any of them, make public